tensorflow_datasets/image/mnist_test.py

```diff
@@ -51,9 +50,23 @@ class KMNISTTest(MNISTTest):
   DATASET_CLASS = mnist.KMNIST
 
 
+_emnist_test_builder_config = mnist.EMNIST.BUILDER_CONFIGS = [
+  mnist.EMNISTConfig(
+    name="test",
+    class_number=200,
+    train_examples=10,
+    test_examples=2,
+    description="EMNIST test data config.",
+    version="1.0.1",
+  ),
+]
+
+
 class EMNISTTest(MNISTTest):
   DATASET_CLASS = mnist.EMNIST
-  BUILDER_CONFIG_NAMES_TO_TEST = ["test"]
+  DATASET_CLASS.BUILDER_CONFIGS = _emnist_test_builder_config
+  DL_EXTRACT_RESULT = "gzip.zip"
```

Comment:
We prefer to have the fake examples unzipped (at least 1 level). You can use DL_EXTRACT_RESULT to point to the right files. See how MNIST does it.

Reply:
But when I downloaded the `gzip.zip` file, the download manager creates `gzip` folder that is include main files. So I need to extract one more time. If I understand it correctly `DL_EXTRACT_RESULT` changing the `dl_manager.download_and_extract(URL)` result. And `DL_EXTRACT_RESULT` want to str type not dict. It's given an error `TypeError: join() argument must be str or bytes, not 'set'`. So I didn't like MNIST. Is there a point I can't see. I'll be happy to fix this problem. Thank you.